Mimir.Worker/Handler/AddressHandlerMappings.cs

```diff
@@ -8,6 +8,8 @@ public static class AddressHandlerMappings
 {
     public static readonly Dictionary<Address, IStateHandler<StateData>> HandlerMappings = new();
 
+    public static readonly string NCGCurrencyAddress = "0xd6663d0EEC2a7c59FF3cfe3089abF8FEd383227D";
```

Comment:
Could you leave a comment, how this address was derived?

Reply:
Like this https://github.com/planetarium/mimir/issues/180